content/applications/sales/subscriptions/payment_providers/sdd.rst

```diff
@@ -0,0 +1,214 @@
+invoice number), or :guilabel:`Based on Customer ID` to use the customer identifier. With either
+method selected, communication is given to the customer, once they choose the |sdd| payment method.
+
+The configuration options listed under the :guilabel:`Availability` section are used to set payment
+restrictions, and are optional. Use the :guilabel:`Maximum Amount` field to set a maximum payment
+value, and the :guilabel:`Countries` field to restrict payment from any particular countries, if
+desired.
+
+.. important::
+   The :guilabel:`Currencies` field should be left with the default :guilabel:`EUR` tag to ensure
+   |sdd| is only accessible when customers pay with Euros.
+
+In the :guilabel:`Messages` tab, the :guilabel:`Pending Message` field can be used to add a message
+that will be displayed if the order remains pending after the payment is processed. By default,
+this field contains the default message: :guilabel:`Please make an unique payment transfer to
+confirm the SEPA mandate.`
+
+.. image:: sdd/sdd-payment-provider.png
+   :align: center
+   :alt: The SEPA Direct Debit Payment Provider form.
+
+Once all necessary configuration settings are complete, the :icon:`fa-eye-slash`
+:guilabel:`Unpublished` smart button located at the top of the page automatically changes to
+:icon:`fa-globe` :guilabel:`Published` to show the |sdd| payment option is now visible to customers.
+Only keep the payment method published if payments are ready to be received.
+
+.. tip::
+   It is possible to keep the payment method enabled, but **not** visible to customers, until
+   payments are ready to be received. To do this, set the smart button to :icon:`fa-eye-slash`
+   :guilabel:`Unpublished` after setting the :guilabel:`State` to :guilabel:`Enabled`.
+
+.. note::
+   The |sdd| configuration menu can also be accessed by navigating to :menuselection:`Settings app
+   --> Website --> Shop --> Configure SEPA Direct Debit`.
+
+Test mode
+~~~~~~~~~
+
+The |sdd| payment method can be tested before it is accessed by customers. First, ensure the test
+creditor identifier is :ref:`configured <subscriptions/sepa-sdd-configuration>` in **Settings**
+before continuing.
+
+.. warning::
+   It is recommended to use the :guilabel:`Test Mode` setting on a duplicate (or test) database to
+   avoid potential issues with invoice numbering.
+
+Next, select the :guilabel:`Test Mode` option in the :guilabel:`State` field on the form. Upon
+selecting :guilabel:`Test Mode`, an orange :guilabel:`Test mode` banner appears in the upper-right
+corner of the form.
+
+.. note::
+   By default, the payment provider remains :guilabel:`Unpublished` in :guilabel:`Test Mode`, so it
+   is not visible to visitors.
+
+Navigate to :menuselection:`Sales --> Orders --> Quotations` and click the :guilabel:`New` button
+to create a new sales order. Select or create a fake customer, or add yourself, in the field next to
```

Comment:
You're right that the customer is fake but we don't need to mention that, haha. It's enough to tell the reader to fill out the Customer field
```suggestion
to create a new sales order. Select a
```